modules/basic-skills/src/actions/call_api.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios')
+
+/**
+ * @hidden true
+ */
+const callApi = async (url, method, body, memory, variable) => {
+  const client = axios.create({
+    baseURL: url
+  })
+
+  let response = undefined
+  if (method === 'get') {
+    response = await client.get('/').then(res => res.data)
+  } else if (method === 'post') {
+    response = await client.post('/', body).then(res => res.data)
+  } else if (method === 'put') {
+    response = await client.put('/', body).then(res => res.data)
+  } else if (method === 'delete') {
+    response = await client.delete('/').then(res => res.data)
+  }
```

Comment:
FYI, you can pass an object to axios without needing multiple if/else statements:  axios({ method: 'get|post|', url: '', data: '' })